src/react/components/pages/predict/predictResult.tsx

```diff
@@ -25,9 +25,10 @@ export default class PredictResult extends React.Component<IPredictResultProps,
         const { tags, predictions } = this.props;
         const tagsDisplayOrder = tags.map((tag) => tag.name);
         for (const name of Object.keys(predictions)) {
-            const prediction = predictions[name];
-            prediction.fieldName = name;
-            prediction.displayOrder = tagsDisplayOrder.indexOf(name);
+            const prediction = predictions[name] || {};
+            prediction["fieldName"] = name;
+            prediction["displayOrder"] = tagsDisplayOrder.indexOf(name);
+            predictions[name] = prediction;
         }
         // not sure if we decide to filter item by the page
         const items = Object.values(predictions).sort((p1, p2) => p1.displayOrder - p2.displayOrder);
```

Comment:
We could just filter out the predictions with null value and not display them instead of displaying them as NaN. Which do you think is better?

Reply:
filter them out may be better